sites.json

```diff
@@ -3939,6 +3959,18 @@
         ]
     },
 
+    {
+        "name": "Mobills",
```

Comment:
I'm confused, there is already a Mobills entry above, if you take a look. This one not only is duplicated, but will also make the jsonlint fail.

Reply:
Whoops, took this one from the original; looks like this was meant to be an update, not a new entry. I'll fix that shortly.